samples/csharp_dotnetcore/13.core-bot.tests/CoreBot.Tests.csproj

```diff
@@ -24,7 +24,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="6.0.7" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.22.3" />
```

Comment:
since CoreBot already references integration we dont need to dupe this reference, as this wasn causing a package downgrade warning